api/get.js

```diff
@@ -119,6 +119,19 @@ Api.prototype = {
     },
     
     createUrl: function(url, req) {
+        // Ensure the url path is (REST) complete
+        url += _.chain(req.params)
+            .map(function(value, key) {
+                switch(key) {
+                    case 'key':
+                    case 'val':
+                        return '/' + value;
+                    default:
+                        return '';
+                }
+            })
+            .value().join('');
+
```

Comment:
This section is not needed and it might break how apimock should work. If ***key*** and ***val*** are needed just add them a variables (```{{variable}}```)